editor/editor_run.cpp

```diff
@@ -272,6 +276,11 @@ void EditorRun::stop() {
 		}
 	}
 
+	if (DebugAdapterProtocol::get_singleton()->is_active()) {
```

Comment:
This can probably be handled internally by listening to EditorNode "stop_pressed" signal.